### ISSUE TITLE
Make all skill weapons have perfect accuracy

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#29]
Really, we gotta ask ourselves, why are all weapons inaccurate?

Random inaccuracy puts a limiter on the skill any player can achieve. If some shots that would've hit, instead miss, and some shot that would've missed, instead do hit, you end up with a certain area around a player aiming at which *might* result in a hit, but really you just don't know. This obviously means that achieving true aiming skill is impossible. Further so if you realize that all bullets - even HS bullets - take time to travel, and so even a perfectly aimed shot (not in the "might hit" zone) might still end up in that zone due to the target dodging it, and vice versa.

I propose all skill weapons (HS, Flintlock, Rocket, possibly all other snipers) should have perfect accuracy.

**Comment by Gregory-Y** [comment]
I would suggest making them at least have first shot accuracy which is then re-established after about 0.8 seconds. That way people don't spam HS at its firecap. But yeah good suggestion.

**Comment by Hippo856** [comment]
I agree with @NOTGregory - FSA is a good idea, but this idea in general unbalances the game in unrealistic ways.

**Comment by deferred-impact** [comment]
I fail to see how it would create a huge imbalance. HS is poweful, but not that powerful. Even top HS players fallback to some spam gun when trying to get top KPG. All this suggestion does is allow HS players to unleash their full potential - and they're already 90% of the way there anyway

First shot accuracy is a good variation, though. But only if there is at least some UI which shows the FSA cooldown, at best a firing mode ("single") which restricts the firerate to FSA

**Comment by deferred-impact** [comment]
Well the effort here depends on the solution chosen, but the most obvious and easiest one - set the accuracy stat to super high value - is gonna take 5 minutes unironically
